modules/ROOT/pages/usage-metrics.adoc

```diff
@@ -50,11 +50,12 @@ a|If an API is governed, all versions of that API are considered one governed AP
 
 |Message Queue	
 |Anypoint MQ API request: A request made to retrieve one or more messages from the Anypoint MQ APIs 
-|Each Anypoint MQ API request includes up to 100 KB of data. An Anypoint MQ API request over 100 KB counts as multiple requests with no fractional units. Anypoint MQ API requests are calculated in the aggregate across all environments (including production, pre-production, sandbox, and design). Anypoint MQ API requests are currently available only via API and aren’t aggregated on usage reports.
+|Each Anypoint MQ API request includes up to 100 KB of data. An Anypoint MQ API request over 100 KB counts as multiple requests with no fractional units. Anypoint MQ API requests are calculated in the aggregate across all environments (including production, pre-production, sandbox, and design).
```

Comment:
There is no limitation on the size of the message for API requests. 
as per https://docs.mulesoft.com/mq/mq-usage can we adjust the above something in the lines of - 

Anypoint MQ uses API requests to calculate billing. Anypoint MQ API requests are calculated in the aggregate across all environments (including production, pre-production, sandbox, and design).